swarm/pss/protocol.go

```diff
@@ -233,6 +243,21 @@ func (self *Protocol) AddPeer(p *p2p.Peer, topic Topic, asymmetric bool, key str
 	return rw, nil
 }
 
+func (self *Protocol) RemovePeer(asymmetric bool, key string) {
```

Comment:
if the test is the only consumer of this method - does it have to reside within the production code?

Reply:
It won't be. It will be used for connection control wrapper code in implementations.